File: duplicate_ele_34.pl
#!/usr/bin/perl 

## dupicate elemnet and their count ##
print("Enter a number separated by space: ");
$num=<STDIN>;
chomp ($num);

@number=split(/\s+/,$num);
for $i (@number){
	$num_hash{$i}++;
}
#foreach $ele (keys %num_hash){
#	push @{ $group{$num_hash{$ele}} },$ele;
#}
foreach $count (keys %num_hash){
	if($num_hash{$count} ne 1){
		print("$count:-$num_hash{$count}\n");
	}
}

File: PYTHON/duplicate_ele_34.py
#!/usr/bin/env python3
####################################################
# Author: Arun Kumar
# Filename: duplicate_ele_34.py
# Date: 06-feb-2026
# Description: Find duplicate elements and their count from space-separated input
#######################################################

# Get input from user
num = input("Enter a number separated by space: ")

# Split input and count occurrences
numbers = num.split()
num_hash = {}

for i in numbers:
	num_hash[i] = num_hash.get(i, 0) + 1

# Print duplicates (elements with count > 1)
for count in num_hash:
	if num_hash[count] != 1:
		print(f"{count}:-{num_hash[count]}")
